src/lib.rs

```diff
@@ -204,11 +204,14 @@ mod codegen;
 mod parser;
 mod value;
 mod error;
+mod util;
 
 pub mod short;
 pub mod object;
+pub mod number;
 
 use short::Short;
+use number::Number;
```

Comment:
Sounds good. Just wanted to make sure that I mentioned it in case you wanted something different.